debezium-core/src/main/java/io/debezium/relational/TableSchemaBuilder.java

```diff
@@ -134,6 +135,13 @@ public TableSchema create(TopicNamingStrategy topicNamingStrategy, Table table,
             addField(keySchemaBuilder, table, column, null);
             hasPrimaryKey.set(true);
         });
+        if (AbstractRegexTopicNamingStrategy.class.isAssignableFrom(topicNamingStrategy.getClass())) {
```

Comment:
@harveyyue I am not a fan of having the code placed here. Can't we provide a method on the naming strategy that will be called so it can do necessary changes?

Reply:
@jpechane  These code changes will depend on connect-api, so I didn't provide a mehod in TopicNamingStrategy interface.

Reply:
@harveyyue I see, still there can be default no-op method on the interface itself. Maybe we could pass a callback lambda/method handle that would be called.

Reply:
@jpechane I have implemented it, please help to review again.